doc/development/building.rst

```diff
@@ -0,0 +1,16 @@
+Building Applications
+=====================
+
+`Jenkins <https://tssw-ci.lsst.org/>`_ is the chosen Continuous Integration platform.
+Each application should have a build project in Jenkins.
+These applications should then have a dedicated build server.
+
+Each build should run the unit tests.
+If the unit tests pass, the application should generate the deployable package and feed into the deployment system.
+Puppet is the Deployment application.
+See `Puppet Server Installation <https://confluence.lsstcorp.org/display/~avillalo/Puppet+Server+Installation>`_ and https://puppet.com/ for more information.
+
+The packaging system is still being investigated, but is leaning towards using RPMs, in general.
```

Comment:
I think you can reword this paragraph to reflect the current reality:

> Artifacts should be packaged with one of the following adopted solutions:
> * Python software should be packed with conda.
> * RPM is a acceptable for C++ and other types of applications (e.g. the middleware).
> * Java code should use maven. (not sure this is true, maybe check with Dave what he does for Tony).
> * Labview is still TBD.
> * For exceptions an RFC ticket is required.

Reply:
Good suggestion, thanks. Done.